SampleApp/SecondViewController.swift

```diff
@@ -108,5 +109,30 @@ class SecondViewController: UIViewController, UITextFieldDelegate, CLLocationMan
   func locationManager(manager: CLLocationManager, didFailWithError error: NSError) {
     print(error)
   }
+  
+  //Mapview Delegate
+  func mapView(mapView: MKMapView, viewForAnnotation annotation: MKAnnotation) -> MKAnnotationView? {
+    if annotation.isKindOfClass(MKUserLocation) {
```

Comment:
Indeed. When user location is enabled on a map, the blue pulsing "pin" is also an annotation, and so for most purposes you want to filter that out.